articles/deliverability/bounce-classification-codes.md

```diff
@@ -15,6 +15,7 @@ Following is a list of bounce classification codes and their meanings:
 | 23             | Too Large               | The message bounced because it was too large for the recipient.             | Soft         |
 | 24             | Timeout                 | The message timed out.                                                      | Soft         |
 | 25             | Admin Failure           | The message was failed by SparkPost's configured policies.                  | Admin        |
+| 26             | Smart Send Suppression  | The message was suppressed by Smart Send policy.                            | Admin        |
```

Comment:
minor nitpick....I think the bounce class has suppression lower-case in other places.  do we care?